### PR TITLE
[Operators] Support SigmoidCrossEntropyWithLogits

### PR DESCRIPTION
**Pull request by artemrakhov**

### CODE REVIEW

File: include/glow/Graph/Graph.h [hunk 1]
```diff
@@ -268,6 +268,10 @@ class Function final : public Named {
   RegressionNode *createRegression(llvm::StringRef name, NodeValue input,
                                    NodeValue expected);
 
+  SigmoidCrossEntropyWithLogitsNode *
```
Comment (jfix71) [inline include/glow/Graph/Graph.h]: Please add doxygen comments

Comment (artemrakhov) [reply include/glow/Graph/Graph.h]: Done.

**Comment by beicy** [comment]
I only found this op in Caffe2. Does ONNX have the corresponding op?

**Comment by artemrakhov** [comment]
@beicy : no, there's no such op in ONNX

**Pull request merged**